app/src/main/java/ru/vasic2000/fakeweather/MainActivity.java

```diff
@@ -28,4 +72,38 @@ public void onClickBack(View view) {
     public void onClickGear(View view) {
         setContentView(R.layout.activity_setting);
     }
+
+    public void sityPovarovo(View view) {
+        presenter.setSityName("@string/povarovo");
+        setContentView(R.layout.activity_main);
+    }
+
+    public void sityShtutgard(View view) {
+        presenter.setSityName("@string/yalta");
+        setContentView(R.layout.activity_main);
+    }
+
+    public void sityMoscow(View view) {
+        presenter.setSityName("@string/yalta");
+        setContentView(R.layout.activity_main);
+    }
+
+    public void sityMytichshy(View view) {
+        presenter.setSityName("@string/yalta");
+        sityName.setText(presenter.getSityName());
+        setContentView(R.layout.activity_main);
+    }
+
+    public void sityYalta(View view) {
+        presenter.setSityName("@string/yalta");
+        sityName.setText(presenter.getSityName());
+        setContentView(R.layout.activity_main);
+    }
+
+    public void sityStokholm(View view) {
+        presenter.setSityName("@string/stokholm");
+        sityName.setText(presenter.getSityName());
+        setContentView(R.layout.activity_main);
+    }
+
```

Comment:
```suggestion
private void setCity(string city){
presenter.setSityName(city);
sityName.setText(presenter.getSityName());
}
```

И потом во всех обработчиках кнопок вызываем этот метод